doc/develop/test/coverage.rst

```diff
@@ -151,5 +151,16 @@ toolchain and require a different board::
 
 which produces a report in the same location as non-unit testing.
 
+Using LLVM Toolchain
+====================
+
+If you have ``ZEPHYR_TOOLCHAIN_VARIANT=llvm`` set you'll need to use `llvm-cov`
+as your coverage tool. The tool requires some arguments which cannot be passed
+to twister via the command line. To help with that, there exists
+``scripts/utils/llvm-gcov.sh`` in the Zephyr tree. To use::
+
+    $ twister --coverage -p unit_testing -T tests/unit --coverage-tool lcov \
+      --gcov-tool ${ZEPHYR_BASE}/scripts/utils/llvm-gcov.sh
```

Comment:
thanks for adding description, but please provide a way that also works on windows.

Twister is also supported on Windows.

Reply:
Done